src/index.ts

```diff
@@ -100,7 +100,7 @@ const getOptionalInput = (name: string) => core.getInput(name) || undefined;
       return;
     }
     case hasChangesets && !hasNonEmptyChangesets:
-      console.log("All changesets are empty; not creating PR");
+      core.warning("All changesets are empty; not creating PR");
```

Comment:
```suggestion
      core.info("All changesets are empty; not creating PR");
```

Reply:
This is a "normal situation" so I think that it shouldn't be a warning. Happy to discuss this further if you feel differently though.

Reply:
No, that makes sense, thanks for the fix.